pipelines.go

```diff
@@ -115,12 +115,17 @@ type PipelineTestCases struct {
 	Classname      string         `json:"classname"`
 	File           string         `json:"file"`
 	ExecutionTime  float64        `json:"execution_time"`
-	SystemOutput   string         `json:"system_output"`
+	SystemOutput   SystemOutput   `json:"system_output"`
 	StackTrace     string         `json:"stack_trace"`
 	AttachmentURL  string         `json:"attachment_url"`
 	RecentFailures RecentFailures `json:"recent_failures"`
 }
 
+type SystemOutput struct {
```

Comment:
Please add a simple comment, like `// SystemOutput describes the pipeline test results.`